nomad/leader_test.go

```diff
@@ -3,6 +3,7 @@ package nomad
 import (
 	"errors"
 	"fmt"
+	"github.com/shoenig/test/must"
 	"sort"
 	"strconv"
 	"testing"
```

Comment:
Are you using GoLand? I swear recently stopped automatically correctly sorting imports by stdlib vs. 3rdparty